pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
+
+- Start a presentation from a markdown file:
+
+`mdp {{presentation.md}}`
+
+- Disable fading transitions:
+
+`mdp --nofade {{presentation.md}}`
+
+- Invert colors for light background:
```

Comment:
I'm afraid that would give the idea that **mdp** changes the background color (and inverts *every* color, including the background), but it doesn't, it only changes the text color. Maybe removing the *invert* thing from the description? like: "Use dark font color for light background terminals".